Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -955,7 +955,14 @@ public async Task OnPlaybackStopped(PlaybackStopInfo info)
 
                     if (libraryItem is IHasMediaSources)
                     {
-                        mediaSource = await GetMediaSource(libraryItem, info.MediaSourceId, info.LiveStreamId).ConfigureAwait(false);
+                        try
+                        {
+                            mediaSource = await GetMediaSource(libraryItem, info.MediaSourceId, info.LiveStreamId).ConfigureAwait(false);
```

Comment:
just catching the exception here, does not _solve_ the problem.
I would rather look into _why_ its throwing, it has have a reason.